specification/cognitiveservices/OpenAI.Inference/models/embeddings_create.tsp

```diff
@@ -1,11 +1,23 @@
 import "@typespec/rest";
 import "@typespec/http";
+import "@typespec/versioning";
 
 using TypeSpec.Rest;
 using TypeSpec.Http;
+using TypeSpec.Versioning;
 
 namespace Azure.OpenAI;
 
+@doc("The format used to represent embeddings.")
+@added(ServiceApiVersions.v2023_12_01_Preview)
```

Comment:
Was this purely a December API gap or has it been longstanding? If we've been missing this longer, I think we should retroactively add it as far back as is accurate (I don't think we should view it as a breaking change given it's fixing a missing component). If we want to introduce it at an intermediate point for full safety and due diligence, it'd probably make sense to stage this as a 2024-02-15-preview (/ 2024-02-01, dependent) addition.